oeps/processes/oep-0055-proc-project-maintainers.rst

```diff
@@ -221,18 +227,23 @@ Coordination will typically be over asynchronous channels.
 
 From time-to-time, a task force may be required to complete certain
 project-wide responsibilities like developing repository standards,
-and tools for measuring compliance to those standards.
+as well as tools for measuring compliance to those standards.
 
 Becoming a Maintainer
 ---------------------
 After the bootstrapping phase, the Open edX project will need
 processes for adding maintainers, replacing maintainers, and ensuring
 that new components are maintained.
 
-**This section should be congruent with the Core Contributor process**
-
 A new maintainer can be added in any of the following ways:
 
+..
+  Should this follow CC procedure (nomination, public discussion?)
+  Is there reason for this to be private/within the scrum of scrums?
+  Is there any reason to call upon the `program administrators`_,
+  maybe to be a backstop or confer access priviledges if the original
+  maintainers are all gone?
```

Comment:
It's very difficult to think this through without some concrete examples of repos which no longer have a maintainer. I even have some trouble at this stage imagining when a new maintainer would be added. I'm assuming in cases where a maintainer team is in effect that adding people to their maintainer team would be an internal process, and not something we need to consider here. Is there a case for adding new maintainers *except* when the old one is gone? Possibly a handoff type situation.

All of this would be easier to imagine with a draft list of maintainers so maybe I just need to catch up on whether that exists yet.

Reply:
@hurtstotouchfire OEPs are not immutable. I really encourage authors and reviewers to consider mostly the happy path, along with the very most likely scenarios. Trying to "future proof" the document to consider all cases makes the document extremely hard to approach. We can always add to the OEP if some corner cases start popping up repeatedly.